src/shared/mem_op.c

```diff
@@ -109,7 +109,7 @@ char *os_LoadString(char *at, const char *str)
         }
         at = newat;
 
-        strncat(at, str, strsize);
+        strncat(at, str, strsize + 1);
```

Comment:
Hi @jnasselle 

Yes, `strncat` guarantees the null-terminated string, I will implement the functionality with `strcat` because the dst buffer always has one byte more than the src.

Best regards
Hanes